src/lando/ui/jinja2/partials/layout.html

```diff
@@ -1,3 +1,11 @@
+{% if request %}
```

Comment:
I'm very surprised there is no way to add our own custom variables/functions/etc to templates via Python.

Reply:
> I'm very surprised there is no way to add our own custom variables/functions/etc to templates via Python.

There is, it's just that the request context is not readily available for those variables/functions. Technically you could implement a method that gets passed the request and call it here (e.g., `user_is_authenticated = check_if_user_is_authenticated(request)`).

In the case of `user_is_authenticated`, the variable is very simple and more so used like an alias, and doesn't really need a separate method.

Reply:
I guess implementing that method, even if it seems kind of redundant, will remove the need to include an if statement in the template. So upon rethinking this, I think it's OK to do that here.

Reply:
Filed [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1909734) to track the improvement to this.